input/pagecontent/reference.md

```diff
@@ -2117,7 +2117,7 @@ In contrast with other names in FSH (for profiles, extensions, etc.), alias name
 **Examples:**
 
   ```
-  Alias: SCT = http://snomed.info/sct
+  Alias: $SCT = http://snomed.info/sct
  
   Alias: $RaceAndEthnicityCDC = urn:oid:2.16.840.1.113883.6.238
```

Comment:
On 2293 this Alias is used without the `$`.

Reply:
Also 2248:
```
  * extension[ombCategory].valueCoding from OmbEthnicityCategories (required)
```
to 
```
  * extension[ombCategory].valueCoding from $OmbEthnicityCategories (required)
```
and 2251
```
  * extension[detailed].valueCoding from DetailedEthnicity (required)
```
to
```
  * extension[detailed].valueCoding from $DetailedEthnicity (required)
```

Reply:
Unless we are supposed to be assuming that those valuesets are defined elsewhere? That's slightly unclear to me.

Reply:
OmbEthnicityCategories is a name of a value set in US Core. So that is not an alias.

Reply:
Same with DetailedEthnicity.

Reply:
I added comments to indicate these are value sets defined in US Core.